test.disabled/ejabberd_tests/tests/mod_global_distrib_SUITE.erl

```diff
@@ -371,6 +372,26 @@ test_component_on_one_host(Config) ->
 
     [escalus:fresh_story(Config, [{User, 1}], Story) || User <- [alice, eve]].
 
+%% Ensures that 2 components in distinct data centers can communicate.
+test_components_in_different_regions(_Config) ->
+    ComponentCommonConfig = [{host, <<"localhost">>}, {password, <<"secret">>},
+                             {server, <<"localhost">>}, {component, <<"test_service">>}],
+    Component1Config = [{port, 8888}, {component, <<"service1">>} | ComponentCommonConfig],
+    Component2Config = [{port, 9990}, {component, <<"service2">>} | ComponentCommonConfig],
+
+    {Comp1, Addr1, _Name1} = component_SUITE:connect_component(Component1Config),
```

Comment:
`connect_component ` should into `component_helper`. Not in this PR.

Reply:
I've created a dedicated task for this.